spec/spec_helper.rb

```diff
@@ -59,6 +59,7 @@
 
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  config.include Capybara::DSL
```

Comment:
This is required since we removed all the `type: :feature` jazz above.

Reply:
maybe indicate this in a comment?

Reply:
@ndushay https://github.com/sul-dlss/infrastructure-integration-test/pull/515